Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -99,7 +103,19 @@ void WiiPane::CreateMisc()
   auto* misc_settings_group_layout = new QGridLayout();
   misc_settings_group->setLayout(misc_settings_group_layout);
   m_main_layout->addWidget(misc_settings_group);
-  m_pal60_mode_checkbox = new QCheckBox(tr("Use PAL60 Mode (EuRGB60)"));
+
+  auto* refresh_rate_pal_box = new QGroupBox(tr("Refresh Rate (PAL)"));
+  auto* refresh_rate_pal_layout = new QHBoxLayout;
+
+  refresh_rate_pal_box->setLayout(refresh_rate_pal_layout);
+  m_refresh_rate_pal_mode_50hz = new QRadioButton(tr("50 Hz"));
+  m_refresh_rate_pal_mode_60hz = new QRadioButton(tr("60 Hz"));
+
+  // refresh_rate_pal_layout->addStretch(1);
```

Comment:
Please don't leave commented out code lying around like this. Either remove it completely, or use it.